packages/api/src/plugin/tokens.ts

```diff
@@ -31,6 +31,6 @@ export const commonTokens = Object.freeze({
  * ```
  * @param tokens The tokens as args
  */
-export function tokens<TS extends string[]>(...tokens: TS): TS {
-  return tokens;
+export function tokens<TS extends string[]>(...tokensList: TS): TS {
```

Comment:
Why is this name change needed? An eslint rule?

Reply:
no-shadow :) but only parameter name is changed so i guess it doesnt break anything :)